selfdrive/ui/qt/offroad/networking.cc

```diff
@@ -278,7 +278,7 @@ void WifiUI::refresh() {
       QPushButton *forgetBtn = new QPushButton("FORGET");
       forgetBtn->setObjectName("forgetBtn");
       QObject::connect(forgetBtn, &QPushButton::clicked, [=]() {
-        if (ConfirmationDialog::confirm("Forget WiFi Network \"" + QString::fromUtf8(network.ssid) + "\"?", this)) {
+        if (ConfirmationDialog::confirm("Forget Wi-Fi Network \"" + QString::fromUtf8(network.ssid) + "\"?", this)) {
```

Comment:
I think WiFi looks better. what do you think @pd0wm?

Reply:
> Wi-Fi is a proper noun and a registered trademark. Capitalize and hyphenate when referring specifically to Wi-Fi technologies. Don’t use WiFi, wifi, or Wifi.

https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/wi-fi

Reply:
[Wi-Fi](https://www.wi-fi.org/) is technically correct, the best kind of correct.

Reply:
Yes :)

Reply:
alright